### DISCUSSION TITLE
Length of USB to camera matters with PoE?!

### DISCUSSION THREAD

**Discussion by donsandrop** [gtxaspec/wz_mini_hacks#468] [Q&A]
Finally got around to getting ethernet run to my cameras. Tested out the PoE adapters to make sure they all worked, did similar with the USB to micro usb adapters so I can just hook up my cameras and go. 

After what I thought were cable issues or configuration issues come to find out the length of the USB cable to the camera makes a difference. 

I use the [uctronics adapter](https://www.amazon.com/dp/B0B9ZLVGJ4?psc=1&ref=ppx_yo2ov_dt_b_product_details) for my PoE then I use a [USB micro to USB A adapter](https://www.amazon.com/dp/B07GJJLNX7?psc=1&ref=ppx_yo2ov_dt_b_product_details) and then I connect to the 10' (approx) supplied Wyze cable. Unfortunately this never connects unless I switch to the 6" Wyze cable. 

I tested multiple configurations, cables, cameras and all had the same results. 
Happy to be doing something wrong if anyone has any suggestions or tips. 

Thanks!

**Comment by KevinKadow** [comment]
When you say "_this never connects_" do you mean the camera never boots, or the ethernet interface never comes up?

Be aware that many cheap Micro-USB cables **do not pass through the data lines**, and thus ethernet will never come up when using these cables.   If you are looking to hardwire to USB (not just supply power) you need a 'sync' cable.

**Comment by bjs-pdx** [comment]
> I use the [uctronics adapter](https://www.amazon.com/dp/B0B9ZLVGJ4?psc=1&ref=ppx_yo2ov_dt_b_product_details) for my PoE then I use a [USB micro to USB A adapter](https://www.amazon.com/dp/B07GJJLNX7?psc=1&ref=ppx_yo2ov_dt_b_product_details) and then I connect to the 10' (approx) supplied Wyze cable. Unfortunately this never connects unless I switch to the 6" Wyze cable.
>
The cable from Wyze only provides power, not data.  The 10' cable will never work for the Uctronics ethernet adapter.

**Reply by donsandrop** [reply]
@bjs-pdx there are two cables provided by wyze, a short one and a long one.
Can confirm the short provides power and data and works with the Uctronics.

The long one I'm unable to confirm network.

Have tried a [20' "data sync"](https://www.amazon.com/gp/product/B07QPHKNN6/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1) cable and just power not network.
Tried swapping the USB 2.0 A Female to USB Micro Female Adapter Converter with the one noted in this conversation: [#73 comment] (https://github.com/gtxaspec/wz_mini_hacks/discussions/73#discussioncomment-3725237) and still no network with the longer cables just power, works with the short cables. 

Last thing I can try related to cabling is the suggested 10' noted in this conversation: [#73 comment] (https://github.com/gtxaspec/wz_mini_hacks/discussions/73#discussioncomment-3725237).

Unfortunately my theory is cable length plays a role here, or some other PoE related setting.
Any thoughts, happy to hear them.

**Comment by donsandrop** [comment]
The cable that is the suggested 10' noted in this conversation: [https://github.com/gtxaspec/wz_mini_hacks/discussions/73 comment] (https://github.com/gtxaspec/wz_mini_hacks/discussions/73#discussioncomment-3725237) has proved to be working. 
However, the experience I'm having now is the camera will get power and ethernet over the PoE adapter and then eventually lose network.

The camera is working, or so I think because the IR lights are responding to me turning the lights on and off in the room.
However, running a continuous ping eventually the "host is down".
